cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/sidecar/ArtifactLocalizer.java

```diff
@@ -99,7 +99,9 @@
   @Inject
   public ArtifactLocalizer(CConfiguration cConf, RemoteClientFactory remoteClientFactory) {
     this.remoteClient = remoteClientFactory.createRemoteClient(Constants.Service.APP_FABRIC_HTTP,
-                                         HttpRequestConfig.DEFAULT,
+                                                               new HttpRequestConfig(
```

Comment:
Nit: Make a HttpRequestConfig constant. This way it would be easier to refactor later